### DISCUSSION TITLE
How to add commands to the command palette

### DISCUSSION THREAD

**Discussion by richtabor** [WordPress/developer-blog-content#157] [Topic Ideas]
We have [this resource](https://make.wordpress.org/core/2023/07/17/introducing-the-wordpress-command-palette-api/#:~:text=WordPress%206.3%20ships%20with%20a,or%20command%20%2B%20k%20keyboard%20shortcuts.) here, but it's highly technical (as it should be). 

I think having a guide with a few types of commands would be nice. Like a command to toggle on a block editor preference, go to an external help guide article, open a plugin sidebar, etc. Even commands to open up internal admin pages; like a host adding a command to view hosting config, or builders adding a command to open another plugin's admin page, etc.

Not only to show how, but give ideas on what folks can add commands for. Inspire a bit. 

It doesn't have to be complex, perhaps simple like how we write about block style variations. 

What's the easiest way for existing plugins to add commands to the command palette?

**Comment by bph** [comment]
Discussion is locked, please continue the conversation on the issue 

- #164
